runbotics/runbotics-orchestrator-ui/src/src-landing/components/FeaturedBlogCard/FeaturedBlogCard.tsx

```diff
@@ -20,38 +20,40 @@ const FeaturedBlogCard: FC<FeaturedBlogCardProps> = ({ post }) => {
     const { translate } = useTranslations();
     if(!post) return null;
     return (
-        <article className={styles.root}>
-            <div className={styles.content}>
-                <Image
-                    src={post.featuredImage.url}
-                    fill
-                    alt=""
-                    className={styles.img}
-                />
-                <div className={styles.info}>
-                    <CardBadge text={post.category.title} />
-                    <Typography variant="body4">
-                        {moment(post.date).format('D.MM.YYYY')}
-                    </Typography>
-                    <Typography variant="body4" className={styles.category}>
-                        {post.category.title}
-                    </Typography>
-                </div>
-                <Typography variant="h3" className={styles.title}>
-                    {post.title}
-                </Typography>
-                <Typography variant="body3" className={styles.description}>
-                    {cutText(post.summary, 230)}
-                </Typography>
-                <div className={styles.readMore}>
-                    <Link href={`/blog/post/${post.slug}`}>
-                        <Typography variant="body5">
-                            {translate('Landing.Blog.Card.ReadMore')}
+        <div className={styles.root}>
+            <Link className={styles.link} href={`/blog/post/${post.slug}`}>
+                <article className={styles.article}>
+                    <div className={styles.content}>
+                        <Image
+                            src={post.featuredImage.url}
+                            fill
+                            alt=""
+                            className={styles.img}
+                        />
+                        <div className={styles.info}>
+                            <CardBadge text={post.category.title} />
+                            <Typography variant="body4">
+                                {moment(post.date).format('D.MM.YYYY')}
```

Comment:
plx, look for moment appearances in LP and substitute with `new Intl.DateTimeFormat().format()`